source/_integrations/mystrom.markdown

```diff
@@ -29,12 +29,18 @@ There is currently support for the following device types within Home Assistant:
 The myStrom integration allows you to control your [myStrom](https://mystrom.ch/) Wi-Fi Bulbs and Wi-Fi Switches. Make sure that you have enabled the REST API under **Advanced** in the web frontend of the switch.
 
 Supported devices are:
- - Switch CH v1 (101)
- - Bulb (102)
- - LED strip (105)
- - Switch CH v2 (106)
- - Switch EU (107)
- - Switch Zero (120)
+
+- Switch CH v1 (101)
+- Bulb (102)
+- LED strip (105)
+- Switch CH v2 (106)
+- Switch EU (107)
+- Switch Zero (120)
+
+Two new sensors are available for switches:
```

Comment:
The consumption was always available not as a sensor but an attribute. We should simply mention what available.

```suggestion
Two sensors are available for switches:
```